reference-service/src/main/resources/db/migration/common/V3.76__fix_corrupted_uuids.sql

```diff
@@ -0,0 +1,31 @@
+-- DBC
+UPDATE `DBC` SET `uuid` = uuid() WHERE `dbc` = '1-1RSG4X0' and `uuid` = '39cc415c-4164-11';
+
+-- FundingType
+UPDATE `FundingType` SET `uuid` = uuid() WHERE `code` = 'ACADEMIC_UNIVERSITY' and `uuid` = 'e0201436-3116-11';
+
+-- Nationality
+UPDATE `Nationality` SET `uuid` = uuid() WHERE `nationality` = 'Batswana' and `uuid` = '43f29aee-a365-11';
+UPDATE `Nationality` SET `uuid` = uuid() WHERE `countryNumber` = 'Citizen of Seychelles' and `uuid` = '4f5613aa-e988-11';
+
+-- ReligiousBelief
+UPDATE `ReligiousBelief` SET `uuid` = uuid() WHERE `code` = 'Agnostic' and `uuid` = '0f8d7be9-3efb-11';
+UPDATE `ReligiousBelief` SET `uuid` = uuid() WHERE `code` = 'Atheist' and `uuid` = '0f8d8039-3efb-11';
+UPDATE `ReligiousBelief` SET `uuid` = uuid() WHERE `code` = 'Baptist' and `uuid` = '0f8d81a9-3efb-11';
+UPDATE `ReligiousBelief` SET `uuid` = uuid() WHERE `code` = 'Christian' and `uuid` = '0f8d832e-3efb-11';
+UPDATE `ReligiousBelief` SET `uuid` = uuid() WHERE `code` = 'Church of England' and `uuid` = '0f8d83ea-3efb-11';
+UPDATE `ReligiousBelief` SET `uuid` = uuid() WHERE `code` = 'Church of Ireland' and `uuid` = '0f8d8448-3efb-11';
+UPDATE `ReligiousBelief` SET `uuid` = uuid() WHERE `code` = 'Church of Scotland' and `uuid` = '0f8d84ab-3efb-11';
+UPDATE `ReligiousBelief` SET `uuid` = uuid() WHERE `code` = 'Druze' and `uuid` = '0f8d8510-3efb-11';
+UPDATE `ReligiousBelief` SET `uuid` = uuid() WHERE `code` = 'Greek Catholic' and `uuid` = '0f8d85d0-3efb-11';
+UPDATE `ReligiousBelief` SET `uuid` = uuid() WHERE `code` = 'Greek Orthodox' and `uuid` = '0f8d862e-3efb-11';
+UPDATE `ReligiousBelief` SET `uuid` = uuid() WHERE `code` = 'Hindu' and `uuid` = '0f8d868c-3efb-11';
+UPDATE `ReligiousBelief` SET `uuid` = uuid() WHERE `code` = 'Methodist' and `uuid` = '0f8d8be1-3efb-11';
+UPDATE `ReligiousBelief` SET `uuid` = uuid() WHERE `code` = 'Not religious' and `uuid` = '0f8d8f98-3efb-11';
+UPDATE `ReligiousBelief` SET `uuid` = uuid() WHERE `code` = 'Orthodox Christian' and `uuid` = '0f8d9054-3efb-11';
+UPDATE `ReligiousBelief` SET `uuid` = uuid() WHERE `code` = 'Presbyterian' and `uuid` = '0f8d9257-3efb-11';
+UPDATE `ReligiousBelief` SET `uuid` = uuid() WHERE `code` = 'Quaker' and `uuid` = '0f8d9374-3efb-11';
+UPDATE `ReligiousBelief` SET `uuid` = uuid() WHERE `code` = 'Roman Catholic' and `uuid` = '0f8d9682-3efb-11';
+UPDATE `ReligiousBelief` SET `uuid` = uuid() WHERE `code` = 'Scientologist' and `uuid` = '0f8d976d-3efb-11';
+UPDATE `ReligiousBelief` SET `uuid` = uuid() WHERE `code` = 'Shinto' and `uuid` = '0f8d97db-3efb-11';
+UPDATE `ReligiousBelief` SET `uuid` = uuid() WHERE `code` = 'Zoroastrian' and `uuid` = '0f8d9b3a-3efb-11';
```

Comment:
For all those newly generated UUIDs, I will check if Prod and NIMDTA match after these scripts are executed. If they don't match, a following PR will be raised to modify the ones in NIMDTA.